its/ruling/src/test/resources/expected/plsql-ParsingError.json

```diff
@@ -2,6 +2,9 @@
 'project:alexandria-plsql-utils/demos/02_create_temp_directory.sql':[
 4,
 ],
+'project:alexandria-plsql-utils/demos/plsql_status_web_pkg.pkb':[
+42,
```

Comment:
This indicates new parsing errors, that's not good. ;-) See my comment on PlSqlKeyword.java
Examples: [column named "sequence"](https://github.com/mortenbra/alexandria-plsql-utils/blob/bc33845a33b0d31f508e759283acb00c28dbf1a7/demos/plsql_status_web_pkg.pkb#L42) and [parameter "cache"](https://github.com/mortenbra/alexandria-plsql-utils/blob/bc33845a33b0d31f508e759283acb00c28dbf1a7/ora/ftp_util_pkg.pkb#L179) are now failing